### DISCUSSION TITLE
Adding more DBs to project [Mar 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5286] [Discord Threads]
> 🙋‍♂️ **replavar**  
## Is there also a trick to how you add db to a project in the latest timely build?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1071010713419661322/image.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### guess it was only one earlier too ?    
####    
#### however we should have a easter-egg for it. lemme chk  
#  
> 💁 **o1lab**  
#### double click show/hide 🙂  
#  
> 🙋‍♂️ **replavar**  
#### Haha, I start seeing a pattern there  😄 It works, thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1071010713616781362)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
